proto/kvrpcpb.proto

```diff
@@ -894,22 +893,25 @@ message ScanDetailV2 {
     // there are a lot of deleted MVCC keys.
     uint64 total_versions = 2;
 
+    // total number of bytes from Scan or kvGet
+    uint64 read_bytes = 3;
+
     // Total number of deletes and single deletes skipped over during
     // iteration, i.e. how many RocksDB tombstones are skipped.
-    uint64 rocksdb_delete_skipped_count = 3;
+    uint64 rocksdb_delete_skipped_count = 4;
```

Comment:
This is a breaking change and please don't do it. 

https://developers.google.com/protocol-buffers/docs/proto#assigning_field_numbers

> each field in the message definition has a unique number. These numbers are used to identify your fields in the message binary format, and should not be changed once your message type is in use.